grub-core/Makefile.core.def

```diff
@@ -2358,3 +2358,9 @@ module = {
   common = loader/i386/xen_file64.c;
   extra_dist = loader/i386/xen_fileXX.c;
 };
+
+module = {
+  name = fwconfig;
+  common = commands/fwconfig.c;
+  enable = x86;
```

Comment:
Not in this form - it's implemented using port IO, which ARM doesn't have. There's apparently support for a memory mapped implementation, but it'll need some different support.